### DISCUSSION TITLE
Google Webmastertools: Failed: Redirect error when using prefix strategy

### DISCUSSION THREAD

**Discussion by notflip** [nuxt-modules/i18n#1773] [Q&A]
Has anyone else had problems with the prefix strategy and the google crawlers giving an error
```
Failed: Redirect error for the domain https://mysite.com
```
I know, with the prefix that / will redirect to /en, but that shouldn't trigger that response from google.
Has anyone else had problems with this?

**Comment by ChrisGV04** [comment]
I'm having the exact same situation.

I'm based in Mexico and my default language is Spanish for the root path, and the English version is with the path prefix `/en`. I have browser detection for the root path only and that causes issues with Google indexing my site, since it only indexed `/en` and it doesn't detect `/` for the Spanish version even if I have both on my sitemap.

Does someone know if there's a way to avoid redirects for robots and crawlers?

I cannot have my Spanish homepage on Google Search because of the redirects. Those should only happen on browsers, not for site crawlers.

**Reply by FollowJack** [reply]
any updates?
